spec/features/saml/saml_logout_spec.rb

```diff
@@ -144,4 +144,31 @@
       expect(page).to have_current_path(root_path)
     end
   end
+
+  context 'remote logout' do
+    let(:sp) { ServiceProvider.find_by(issuer: saml_settings.issuer) }
+    let(:agency) { sp.agency }
+
+    it "terminates the user's session remotely" do
+      # set up SP identity and agency identity
+      user = sign_up_and_2fa_ial1_user
+      visit_saml_authn_request_url
+      click_continue
+      click_agree_and_continue
+
+      agency_uuid = AgencyIdentity.find_by(user_id: user.id, agency_id: agency.id).uuid
+      # Confirmed that this actually hits the controller action
+      send_saml_remote_logout_request(overrides: { sessionindex: agency_uuid })
+
+      # THIS SHOULD PASS
```

Comment:
We would expect the session to have been cleared at this point since we are going through the remote logout controller action. However, the session is still present (as shown by the next assertion). Manually destroying the redis session at this point does log out the user / cause the test to pass, so I'm pretty sure this code as written would actually fix remote logout in practice.